### ISSUE TITLE
Has Twitter headers expired?

### ISSUE THREAD

**Issue by skywd** [bisguzar/twitter-scraper#165]
Unable to get user tweets and basic information

**Comment by ggdog007** [comment]
same problem here and I tried other similar twitter scrapper without twitter API recently and got None. Maybe they've update something.

**Comment by mckinnonn** [comment]
Seems as though they have completely phased out legacy API. Request just returns a blank response in postman for me.

**Comment by saizoukun** [comment]
https://github.com/bisguzar/twitter-scraper/issues/140

**Comment by TheMulti0** [comment]
Same

**Comment by caligoig** [comment]
Same
Returns 400

**Comment by TheMulti0** [comment]
Seems like it has to do with a change in the Twitter API (v2), I can see that right now you cannot view tweets without logging in.

**Comment by Altimis** [comment]
@TheMulti0  Yes indeed. Here is an alternative without authentification : [scweet](https://github.com/Altimis/Scweet). I'll add more features in he next days (images links ...).

**Comment by TheMulti0** [comment]
> @TheMulti0  Yes indeed. Here is an alternative without authentification : [scweet](https://github.com/Altimis/Scweet). I'll add more features in he next days (images links ...). 

Good job! I might use it as a fallback engine as selenium is slower than the api.

Looks like a neat library!

**Comment by Altimis** [comment]
@TheMulti0  
Yeah that's the only weakness of it. I'll see if I could lunch many drivers simultaneously since it works in background.
